src/annotator/test/guest-test.js

```diff
@@ -204,6 +204,20 @@ describe('Guest', () => {
       });
     });
 
+    describe('on "focusAnnotations" event', () => {
+      it('calls "Guest#._focusAnnotations"', () => {
+        const guest = createGuest();
+        sandbox.stub(guest, '_focusAnnotations').callThrough();
```

Comment:
It would be preferable if this test didn't refer to internal methods, though you also don't to repeat the tests for the "focusAnnotations" event from the host frame. One way to handle this would be to do something similar to `selectAnnotations` where there is a public method for focusing annotations, plus tests that check that this public method is invoked in response to various events. One benefit of this is that it might enable the code that handles events from the host/sidebar frames to be separated into a helper class in future to make the overall Guest class easier to follow.

Reply:
My intent is not to test the `_focusAnnotations` method but make sure that the `focusAnnotations` RPC event triggers the method. I could test the final outcome.